frontend/jupyter/src/app/services/config.ts

```diff
@@ -19,7 +19,7 @@ export function getStopDialogConfig(name: string): DialogConfig {
   return {
     title: $localize`Are you sure you want to stop this notebook server? ${name}`,
     message: $localize`Warning: Your data might be lost if the notebook server
-                       is not backed by persistent storage.`,
+                       is not backed by persistent storage`,
```

Comment:
Why remove the . ?

Reply:
The getDeleteDialogConfig didn't have a dot at the end of the warning message, so I was just making it consitent. Unless you think the solution is to add the dot to the other warning messages.